packages/shared-state/files/www/cgi-bin/shared-state-multiwriter

```diff
@@ -27,3 +27,11 @@ PATH_INFO="${PATH_INFO#/}"
 echo "$PATH_INFO" | grep -q '^[a-zA-Z0-9_-]*$' || exit 22
 
 cat - | shared-state-multiwriter reqsync "$PATH_INFO"
```

Comment:
shouldn't this line go away?

Reply:
Done! Sorry for the delay, I  though that I already did it